### PR TITLE
Adding vagrant

### PR DESCRIPTION
**Pull request by Danielsmithitguy**

### CODE REVIEW

**Comment by gtwalford** [comment]
Thanks! I'll review and test this over the weekend

File: Vagrantfile [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+      nvm install 5.9.0
+
+      npm install -g babel babel-cli
+
+      if [ -d "/home/vagrant/Baus-Boilerplate/node_modules" ]
+      then
+          npm rebuild node-sass
+      else
+          npm install
+      fi
```
Comment (rochestb) [inline Vagrantfile]: what is fi?

Comment (jamesearl) [inline Vagrantfile]: @Razn-Gant ^^

Comment (Razn-Gant) [inline Vagrantfile]: What is it, or what is it doing? @rochestb

**Pull request merged**